infer/src/IR/Struct.ml

```diff
@@ -87,7 +87,7 @@ let pp pe name f
      exported_obj_methods: {@[<v>%a@]}@,\
      annots: {@[<v>%a@]}@,\
      java_class_info: {@[<v>%a@]}@,\
-     dummy: %b@]@,"
+     dummy: %b@]@,\n"
```

Comment:
Sorry to nitpick a one-line change but the `Format` module needs to know that a new line starts here so `\n` needs to be `@\n`. I think `@,\n` can just be `@\n`, or you can fold the `@,` ("cut") into the previous vertical box (where cuts become newlines):
```
    dummy: %b@,@]"
```

Reply:
Found the Python programmer 🤚 ... I made your change; works fine!